docs/develop.md

```diff
@@ -0,0 +1,149 @@
+# Development #
+
+This page would be helpful if you want to change something in condo. It contains core concepts and general knowledge. 
+
+
+## Quick start for development
+
+To launch condo, or any other Keystone + Next app in development mode, use this:
+
+`yarn workspace @app/condo dev`
+
+You can change code of the app and inspect changes in `localhost:3000`
+
+
+## Apps and packages
+
+This project is a monorepo, which is split to apps and packages.
+
+Apps are independent applications and cannot use code from each other.
+
+For example: If you need to create a chat app for your residents - you can create new app `apps/chat`
+
+Good example of application is `apps/condo` - it is Keystone + Next based web application that allows you to manage buildings, tickets and residents
+
+Packages, on the other hand, are internal libraries and can be used in any app
+
+Good example of package is `packages/webhooks` it allows you to add webhooks feature to any of your `apps`
+
+
+## Tech stack
+
+Our primary tech stack is:
+- [ApolloGraphQL](https://www.apollographql.com)
+- [KeystoneJS](https://github.com/keystonejs/keystone-5)
+- [NextJS](https://nextjs.org)
+
+All our apps and packages use this tech stack.
+
+
+## Database and migrations
+
+We primarily use Postgres as our main database. MongoDB can be configured in Keystone, but it is not supported and not guaranteed it will work without any changes in codebase
+
+Our database migration process is based on: https://github.com/keystonejs/keystone/discussions/3067
+
+> Note: `apps/condo` is used as an example, but any Keystone + Next based app can be configured to use this migration process. 
+> Please check `apps/condo/package.json` to see how described commands work on the inside
+
+### How to launch migrations
+
+1. Make sure `DATABASE_URL` env like `postgresql://postgres:postgres@127.0.0.1/main` is set.
+2. Make sure your DB is ready to accept connections or run it by docker `docker-compose up -d postgresdb`.
+3. Install kmigrator dependencies:
+    ```bash
+    # install dependencies
+    python3 -m pip install django
```

Comment:
Would be great to specify concrete package versions here. I have troubles running migrations with the latest ones